universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -270,7 +270,6 @@
       <if variable="URL">
         <choose>
           <if type="article-journal article-magazine article-newspaper bill chapter entry-dictionary entry-encyclopedia legislation paper-conference" match="any">
-            <text prefix="[" suffix="]"/>
           </if>
         </choose>
       </if>
```

Comment:
The macro still exists (currently on lines 268-277: https://github.com/citation-style-language/styles/pull/2176/files#diff-ffb843dd284550f02f5bcd696bb2f081R268 )